src/SDL_image.sym

```diff
@@ -63,26 +63,22 @@ SDL3_image_0.0.0 {
     IMG_SaveTGA_IO;
     IMG_SaveGIF;
     IMG_SaveGIF_IO;
-    IMG_SaveWEBP;
-    IMG_SaveWEBP_IO;
```

Comment:
Why did you remove these two? Bad rebase after https://github.com/libsdl-org/SDL_image/commit/6d8fe18ff227cf071a44622e022499cdae9f2500 ??

Reply:
.. and I hope there aren't any other badnesses after rebasing after https://github.com/libsdl-org/SDL_image/commit/6d8fe18ff227cf071a44622e022499cdae9f2500